src/components/EditProfileComponenets/updateSignInDetails.jsx

```diff
@@ -129,112 +131,116 @@ function UpdateSignInDetailes() {
-                        type='email'
-                        value={currentEmail}
-                        onChange={handleCurrentEmailChange}
-                        required
-                    />
-                </div>
-                <div className='flex flex-col md:flex-row md:space-x-5  items-center p-2'>
-                    <label
-                        htmlFor='currentPassword'
-                        className='md:w-1/6 font-semibold text-white'
-                    >
-                        {t("editProfile.CRUDpassword")}
-                        <span className='text-red-600 font-mono '>*</span>
-                    </label>
-                    <input
-                        className='border-4 focus:border-amber-400 focus:border-4 rounded-md p-2 w-52 bg-white focus:bg-green-300 bg-opacity-50'
-                        type='password'
-                        value={currentPassword}
-                        onChange={handleCurrentPasswordChange}
-                        required
-                    />
-                </div>
-                <div className='flex flex-col md:flex-row md:space-x-5  items-center p-2'>
-                    <label
-                        htmlFor='newPassword'
-                        className='md:w-1/6 font-semibold text-white'
-                    >
-                        {t("editProfile.CRUDnew1")}
-                    </label>
-                    <input
-                        className='border-4 focus:border-amber-400 focus:border-4 rounded-md p-2 w-52 bg-white focus:bg-green-300 bg-opacity-50'
-                        type='password'
-                        value={newPassword}
-                        onChange={handleNewPasswordChange}
-                        minLength='8'
-                    />
-                </div>
-                <div className='flex flex-col md:flex-row md:space-x-5  items-center p-2'>
-                    <label
-                        htmlFor='confirmPassword'
-                        className='md:w-1/6 font-semibold text-white'
-                    >
-                        {t("editProfile.CRUDnew2")}
-                    </label>
-                    <input
-                        className='border-4 focus:border-amber-400 focus:border-4 rounded-md p-2 w-52 bg-white focus:bg-green-300 bg-opacity-50'
-                        type='password'
-                        value={confirmPassword}
-                        onChange={handleConfirmPasswordChange}
-                        minLength='8'
-                    />
-                </div>
-                {error && (
-                    <div className='flex justify-center items-center'>
-                        <p className='text-red-500 border-white w-fit bg-red-300 rounded-lg slide-in'>
-                            {error}
-                        </p>
+        <>
+            <IndexPage title='Edit Profile' />
```

Comment:
shouldn't this be in the profile/[id]/editProfile ? it shouldn't be in componenets